scripts/validators/release-version-validator

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Script to verify that AEMM version is updated to reflect the latest release in:
```

Comment:
Very clean! Is it possible to run this first, then have it trigger `update-versions-for-release` if diffs are detected? Or maybe edit Makefile with a target to do both at once.

I like the automation, but it also seems like it's more manual with understanding all the steps to take during a release to make sure this is all aligned

Reply:
There is definitely scope to automate the release prep further. 
For now, I wanted to keep it manual so we can test this workflow for a few releases, before making Travis commit to the repo.